### ISSUE TITLE
Unable to open settings in OS X

### ISSUE THREAD

**Issue by G07cha** [rhysd/Trendy#8]
Tried latest build and release but on clicking "Settings" button from sidebar nothing happens(debug logs are clear).

**Comment by rhysd** [comment]
OK, I'll check it tonight on my iMac.

**Comment by rhysd** [comment]
I think you should set default behavior for `json` files.  When 'Settings' is clicked, Trendy will open it with default app for `json` file.  But if no app is related to `json` files,  Trendy can't know which app should be used to open `json` file.

In OS X,  please open Finder.app and select json file then set default app for `json` files to your favorite editor.  Then could you try clicking 'Settings' again?

**Comment by rhysd** [comment]
Note: You can see the app's log by `$ ./bin/cli.js --no-detach` because Trendy's process is detached by default.  And you can open Chrome DevTools by the same shortcut key as Chrome.

**Comment by G07cha** [comment]
Okay, I get it, default app for JSON files for me is Brackets IDE so it's open in background. But I'm still think that it isn't good for UI application. What do you think about separate page for settings?

**Comment by rhysd** [comment]
Yes, we should implement settings UI for this.  But I could not take time to implement it.  Opening in external app is just workaround.

**Comment by G07cha** [comment]
If you don't mind I keep this issue open until Settings UI don't be implemented
